device/bgp/bgpsrtesegment.yaml

```diff
@@ -3,47 +3,490 @@ components:
+        type_e:
+          $ref: '#/components/schemas/BgpSrte.SegmentETypeSubTlv'
+        type_f:
+          $ref: '#/components/schemas/BgpSrte.SegmentFTypeSubTlv'
+        type_g:
+          $ref: '#/components/schemas/BgpSrte.SegmentGTypeSubTlv'
+        type_h:
+          $ref: '#/components/schemas/BgpSrte.SegmentHTypeSubTlv'
+        type_i:
+          $ref: '#/components/schemas/BgpSrte.SegmentITypeSubTlv'
+        type_j:
+          $ref: '#/components/schemas/BgpSrte.SegmentJTypeSubTlv'
+        type_k:
+          $ref: '#/components/schemas/BgpSrte.SegmentKTypeSubTlv'
+  
+    BgpSrte.SrMplsSid:
+      description: >-
+        Configuration for SR-MPLS with Label, TC, Bottom-of-Stack and TTL.
+      type: object
+      properties:
+        label:
           description: >-
-            MPLS label
+            Label value in [0, 2^20 -1].
           type: integer
-          default: 0  
-        mpls_tc:
+          minimum: 0
+          maximum: 1048575
+        tc:
           description: >-
-            TBD
+            Traffic class in bits.
           type: integer
+          minimum: 0
+          maximum: 7
+        s_bit:
+          description: >-
+            Bottom-of-Stack bit.
+          type: boolean
+        ttl:
+          description: >-
+            Time To Live. 
+          type: integer
+          minimum: 0
+          maximum: 255
+
+    BgpSrte.SRv6SIDEndpointBehaviorAndStructure:
+      description: >-
+        SRv6 Endpoint Behavior and SID Structure and its optional. 
+        The total of the Locator Block, Locator Node, Function, and Argument
+        lengths MUST be less than or equal to 128.
+      type: object
+      properties:
+        lb_length:
+          description: >-
+            SRv6 SID Locator Block length in bits.
+          type: integer
+          minimum: 0
+          maximum: 128
           default: 0
-        mpls_ttl:
+        ln_length:
           description: >-
-            TBD
+            SRv6 SID Locator Node length in bits.
           type: integer
+          minimum: 0
+          maximum: 128
           default: 0
-        v_flag:
+        func_length:
           description: >-
-            TBD
-          type: boolean
-          default: false
-        ipv6_sid:
+            SRv6 SID Function length in bits.
+          type: integer
+          minimum: 0
+          maximum: 128
+          default: 0
+        arg_length:
+          description: >-
+            SRv6 SID Arguments length in bits.
+          type: integer
+          minimum: 0
+          maximum: 128
+          default: 0
+    
+    BgpSrte.SegmentATypeSubTlv:
+      x-include:
+      - '.#/components/schemas/BgpSrte.SrMplsSid'
+      description: >-
+        Type  A: SID only, in the form of MPLS Label.
+      type: object
+      properties:
+        flags:
+          description: >-
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
```

Comment:
One octet bitmap for flags including V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in